File: app/src/main/java/payments/stone/com/br/desafiomobile/home/HomeView.java
package payments.stone.com.br.desafiomobile.home;

import java.util.List;

import payments.stone.com.br.desafiomobile.model.Product;
import payments.stone.com.br.desafiomobile.views.BaseView;

/**
 * Created by william.gouvea on 9/21/17.
 */

public interface HomeView extends BaseView {
    void showProducts(List<Product> productList);
}

File: app/src/main/java/jb/project/starwarsshoppinglist/home/activity/view/HomeView.kt
package jb.project.starwarsshoppinglist.home.activity.view

import jb.project.starwarsshoppinglist.model.Product

/**
 * Created by Jb on 12/10/2017.
 */
interface HomeView {
    fun loadProductList(productList: ArrayList<Product>)
}